nmostesting/mocks/Node.py

```diff
@@ -376,7 +400,7 @@ def node_sdp(media_type, media_subtype):
             depth=CONFIG.SDP_PREFERENCES["video_depth"],
             sampling=CONFIG.SDP_PREFERENCES["video_sampling"],
             colorimetry=CONFIG.SDP_PREFERENCES["video_colorimetry"],
-            fullrange=CONFIG.SDP_PREFERENCES["video_fullrange"],
+            fullrange=CONFIG.SDP_PREFERENCES["video_range"],
```

Comment:
I agree `video_range` with a value is better, so that we could test `FULLPROTECT` and `NARROW` explicitly as well. Please change `fullrange` jinja template variable to just `range` though.

Reply:
Done. SDP parameter names now used to avoid need to map names.